bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,986 @@
+                    changeSettings(newobj);
+                    break;
+                case "soundPressure":
+                    newobj.addProperty("SoundInfo", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "alarmForwarding":
+                    newobj.addProperty("AlarmForwarding", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "averaging":
+                    newobj.addProperty("averaging", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "errorBars":
+                    newobj.addProperty("ErrorBars", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                /*
+                 * Not supported yet! Ready for implementation.
+                 *
+                 * case "getHistoryFiles":
+                 * getDataFiles();
+                 * break;
+                 */ case "ppm_and_ppb":
+                    newobj.addProperty("ppm&ppb", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                case "nightmode_FanNightOff":
+                    subjson.addProperty("FanNightOff", ohCmd2airqCmd(command.toString()));
+                    newobj.add("NightMode", subjson);
+                    changeSettings(newobj);
+                    break;
+                case "nightmode_WifiNightOff":
+                    subjson.addProperty("WifiNightOff", ohCmd2airqCmd(command.toString()));
+                    newobj.add("NightMode", subjson);
+                    changeSettings(newobj);
+                    break;
+                case "SSID":
+                    JsonElement wifidatael = new Gson().fromJson(command.toString(), JsonElement.class);
+                    if (wifidatael != null) {
+                        JsonObject wifidataobj = wifidatael.getAsJsonObject();
+                        newobj.addProperty("WiFissid", wifidataobj.get("WiFissid").getAsString());
+                        newobj.addProperty("WiFipass", wifidataobj.get("WiFipass").getAsString());
+                        String bssid = wifidataobj.get("WiFibssid").getAsString();
+                        if (!bssid.isEmpty()) {
+                            newobj.addProperty("WiFibssid", bssid);
+                        }
+                        newobj.addProperty("reset", wifidataobj.get("reset").getAsString());
+                        logger.info("air-Q - airqHandler - handleCommand(): dummy to change settings: {}",
+                                newobj.toString());
+                        changeSettings(newobj);
+                    } else {
+                        logger.error("Cannot extract wlan data from this string: {}", wifidatael);
```

Comment:
ERROR should only be used when something very critical goes wrong. Link given to how to work out which logging level should be used, please review all uses of the logger and change them when you have some spare time as I wont comment on them all now.

Reply:
I have reviewed all logging calls and replaced ALL .error with .warn as they are not critical. There is also no .info anymore, just .warn, .debug or .trace.